graphql-builder/src/main/java/com/phocassoftware/graphql/builder/TypeBuilder.java

```diff
@@ -164,22 +168,35 @@ public ObjectType(EntityProcessor entityProcessor, TypeMeta meta) {
 		}
 
 		@Override
-		protected void processFields(String typeName, Builder graphType, graphql.schema.GraphQLInterfaceType.Builder interfaceBuilder)
-			throws ReflectiveOperationException {
+		protected void processFields(String typeName, Builder graphType, graphql.schema.GraphQLInterfaceType.Builder interfaceBuilder) {
 			var type = meta.getType();
-			for (Method method : type.getMethods()) {
-				try {
+			var methods = type.getMethods();
+
+			var validMethods = new HashMap<String, Method>();
+
+			var duplicates = Arrays
+				.stream(methods)
+				.filter(method -> {
 					var name = EntityUtil.getter(method);
-					if (name.isEmpty()) {
-						continue;
-					}
-					var f = entityProcessor.getMethodProcessor().process(null, FieldCoordinates.coordinates(typeName, name.get()), meta, method);
+					if (name.isEmpty()) return false;
+					return validMethods.put(name.get(), method) != null;
```

Comment:
for filters like this. To reduce cognitive load they should all be pure functions